lib/exhi_web/controllers/article_controller.ex

```diff
@@ -0,0 +1,53 @@
+defmodule ExhiWeb.ArticleController do
+  use ExhiWeb, :controller
+
+  def create(conn, _params) do
+    changeset = Exhi.Article.changeset(%Exhi.Article{}, _params["article"])
+    case Exhi.Repo.insert(changeset) do
```

Comment:
I don't believe that the controller should be doing any calls to `Repo`. That should be in the contexts (adding a link in my overview comment).